tests/unit/state/receiver/test_receiver_state.cpp

```diff
@@ -655,3 +718,70 @@ TEST_F(TestReceiverState, RemoveAllReceiverSocketsWorks)
 
   EXPECT_EQ(sacn_remove_receiver_socket_fake.call_count, 2u * SACN_RECEIVER_MAX_UNIVERSES);
 }
+
+TEST_F(TestReceiverThread, AddsPendingSockets)
+{
+  sacn_add_pending_sockets_fake.custom_fake = [](SacnRecvThreadContext* recv_thread_context) {
+    EXPECT_EQ(recv_thread_context, get_recv_thread_context(0u));
+  };
+
+  for (unsigned int i = 1u; i <= 10u; ++i)
+  {
+    iterate_thread(get_recv_thread_context(0u));
+    sacn_add_pending_sockets_fake.call_count = i;
```

Comment:
I noticed a few of these tests iterate and don't seem to do much.  Are you just trying to see that the function is called 10 times, or was there more you were planning to add?

Reply:
The loop is just to demonstrate that the function is called once per iteration and isn't a one-off thing.